<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkPwdWidget</class>
 <widget class="QWidget" name="checkPwdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
border-image: url(:/images/images/passBg2.png);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_title">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frame_title
{
	border-top-left-radius:3px;
	border-top-right-radius:3px;
	background-color:qlineargradient(x1:0,y1:0,x2:1,y2:0,stop:0 rgba(0,83,207,0.9),stop:1 rgba(66,183,255,0.9));
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_title">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_left">
           <property name="minimumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_left">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_logo">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/images/images/logo1.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_title">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>8</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>办公终端人脸识别</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frame_right">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/images/topHang.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_right">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_return">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_return">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton#btn_return
{
	border-image: url(:/images/images/return2.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_min">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_min
{
	border-image: url(:/images/images/min.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_close
{
	border-image: url(:/images/images/close.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>57</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>57</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>66</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>66</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/images/headLeft.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>224</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>224</width>
             <height>37</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(28, 95, 245);</string>
           </property>
           <property name="text">
            <string>验证操作系统密码</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>66</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>66</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/images/headRight.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>135</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>135</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>39</number>
         </property>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_account">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_account">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>操作系统账号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_pwd">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>操作系统密码</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="myPwdLinEdit" name="lineEdit_pwd">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>168</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>168</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_res">
           <property name="minimumSize">
            <size>
             <width>206</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>206</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 76, 120);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>156</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btn">
           <property name="minimumSize">
            <size>
             <width>206</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>206</width>
             <height>48</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#btn
{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
 	border: 2px solid rgb(28, 95, 245); /* 为了更好地显示边框 */
    border-radius: 5px; /* 设置圆角 */
}
QPushButton#btn:hover {
    background-color: rgb(56, 125, 255); /* 鼠标悬浮时改变背景色 */
    border: 2px solid rgb(56, 125, 255); /* 悬浮时边框颜色同步变化 */
}

QPushButton#btn:pressed {
    background-color: rgb(15, 55, 215); /* 按钮被点击时的背景色 */
    border: 2px solid rgb(15, 55, 215); /* 按钮被点击时的边框颜色 */
    padding-top: 2px; /* 可选：让按钮看起来稍微下沉 */
}</string>
           </property>
           <property name="text">
            <string>验证密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myPwdLinEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mypwdlinedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
